Test/PublishProGetUniversalPackage.Tests.ps1

```diff
@@ -165,7 +165,13 @@ function ThenPackagePublished
 
     Assert-MockCalled -CommandName 'Publish-ProGetUniversalPackage' `
                       -ModuleName 'Whiskey' `
-                      -ParameterFilter { $PackagePath -eq (Join-Path -Path $testRoot -ChildPath ('.output\{0}' -f $FileName)) }
+                      -ParameterFilter { 

```

Comment:
Double-assignment through off reading, along with inconsistent use of surrounding parenthesis.